saleor/graphql/order/mutations/orders.py

```diff
@@ -150,6 +150,9 @@ def try_payment_action(order, user, payment, func, *args, **kwargs):
         raise ValidationError(
             {"payment": ValidationError(message, code=OrderErrorCode.PAYMENT_ERROR)}
         )
+    else:
```

Comment:
Not sure if `else` is needed here, errors will raise ValidationError, so you can update total without `else` section.

Reply:
Also, can you double check if you really need this here? It is a wrapper for functions from gateway.py. All actions like capture, refund, etc - are decorated with `gateway_postprocess` which already call `order.update_total_paid`